com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/RestUtils.java

```diff
@@ -170,12 +170,21 @@ public static Query getQuery(String tableName, String whereClause, List<Object>
 			whereClause = whereClause + "AD_Language=?";
 			params.add(Env.getAD_Language(Env.getCtx()));
 		}
+		
+		if(includeInactive!=null && includeInactive.equals("only")) {
+			whereClause += "isActive=?";
+			params.add("N");
+		}
 
 		Query query = new Query(Env.getCtx(), table, whereClause, null);
 
-		query.setApplyAccessFilter(true, false)
-			.setOnlyActiveRecords(true)
-			.setParameters(params);
+		query.setApplyAccessFilter(true, false);
+		
+		if(includeInactive==null)
```

Comment:
This assumes the value of includeinactive parameter can be "only" or anything else indicates yes.

I think at least we could review the values "false", "no", "n", otherwise the users could get confused.

Please also add the documentation at:
https://wiki.idempiere.org/en/REST_Web_Services#Query_Option_includeinactive

Reply:
for this. there is two option to use includeinactive. if user add includeinactive on query, the result will show all record consist active and inactive records. but if user set includeinactive=only on query, the result will show only inactive record without active record.